docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-agent-files-service.md

```diff
@@ -0,0 +1,121 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Configure the agent with file sources
+
+In order for Moderne to obtain information about your organizational structure, you will need to create some configuration files and direct the Moderne agent to said files. This guide will explain how to do both of those things.
+
+## repos.csv
+
+This is a required file, which outlines the repositories and the organizations which they are associated with.
+When provided your organization details, they must be provided in a CSV file with the following format:
+
+| cloneUrl      | branch   | org1    | org2        | org3 |
+|---------------|----------|---------|-------------|------|
+| `https://github.com/openrewrite/rewrite-recipe-bom` | main | Open Source | ALL | |
+| `https://github.com/Netflix/spectator-go` | main | Netflix | Open Source | ALL |
+
+The organizations under `org1`, `org2`, `org3`, etc. represent the hierarchy of organizations. There is no limit to the number of organizations that can be provided via this CSV.
+
+The above example would be used in Moderne DX to generate an organizational listing of the following:
```

Comment:
We say that this could be used in Moderne DX - but wouldn't this also work for the platform? So maybe we should just remove the Moderne DX part?